rust/worker/src/tracing/opentelemetry_config.rs

```diff
@@ -26,24 +70,33 @@ pub(crate) fn init_otel_tracing(service_name: &String, otel_endpoint: &String) {
         .tracing()
         .with_exporter(exporter)
         .with_trace_config(trace_config)
-        .install_batch(opentelemetry::runtime::Tokio)
-        .expect("Error - Failed to create tracer.");
+        .install_batch(opentelemetry_sdk::runtime::Tokio)
+        .expect("could not build otlp trace provider")
+        .tracer(service_name.clone());
     // Layer for adding our configured tracer.
     // Export everything at this layer. The backend i.e. honeycomb or jaeger will filter at its end.
-    let exporter_layer = tracing_opentelemetry::layer()
-        .with_tracer(otlp_tracer)
+    let exporter_layer = tracing_opentelemetry::OpenTelemetryLayer::new(otlp_tracer)
         .with_filter(tracing_subscriber::filter::LevelFilter::TRACE);
     // Layer for printing spans to stdout. Only print INFO logs by default.
     let stdout_layer =
         BunyanFormattingLayer::new(service_name.clone().to_string(), std::io::stdout)
+            .with_filter(tracing_subscriber::filter::FilterFn::new(|metadata| {
+                !(metadata
+                    .module_path()
+                    .unwrap_or("")
+                    .starts_with("chroma_cache")
+                    && metadata.name() != "clear")
```

Comment:
What does clear mean in this context?

Reply:
Clear is the name of the cache clear call.

Reply:
Oh ok, Is there anyway to put this logic in the sampler and have it take effect here so that there is only one place where we do this sort of sampling? 

My concern is that this is really specific and coupled to knowing "clear" etc. And localizing it could help readability. 

It is not a big deal today while this is small, but can become unruly as a pattern.